mlflow/server/js/src/common/components/PageContainer.tsx

```diff
@@ -34,7 +34,7 @@ PageContainer.defaultProps = {
 
 const styles = {
   useFullHeightLayout: {
-    height: '100%',
+    height: 'calc(100% - 60px)', // 60px comes from header height
```

Comment:
aren't we going back and forth with this one? not sure if shouldn't be skipped

Reply:
@daniellok-db yeah I confirm it would be good to revert this one, otherwise few pages get cut:

https://github.com/user-attachments/assets/0969368a-e2bd-4dda-a9c0-cb0f2e8e77c6